packages/lib-classifier/src/plugins/drawingTools/models/marks/Mark/Mark.spec.js

```diff
@@ -192,5 +198,21 @@ describe('Models > Drawing Task > Mark', function () {
       })
     })
   })
+
+  describe('snapshots', function () {
+    let snapshot
+
+    before(function () {
+      snapshot = getSnapshot(mark)
+    })
+
+    it('should not have an ID', function () {
+      expect(snapshot.id).to.be.undefined()
+    })
+
+    it('should have an annotations array', function () {
+      expect(snapshot.annotations).to.be.a('array')
+    })
```

Comment:
Could we test that the annotations are ordered as expected in the outputted snapshot? It makes me nervous that we can't choose how the snapshot serializes maps.

Reply:
I thought I'd added a test for that. Did I only test classifications?

Reply:
The output order of Mark subtasks isn’t important in the final analysis. That’s why I tested classifications but didn’t test these.

Reply:
I've updated this to mock and test a mark with annotations. That said, `mark.annotations` is removed when we serialise the classification (see ADR 25) so what this is testing is that we shouldn't lose any annotations, I guess.

Reply:
My comment has to do with type conversions and we should make sure the type conversion is doing what we expect it to independently in keeping with the points raised in ADR 10. MST isn't really giving us a choice in how the map conversion happens using snapshots so I wanted extra safety checks.